iceoryx_posh/test/moduletests/test_roudi_process_manager.cpp

```diff
@@ -55,7 +55,7 @@ class ProcessManager_test : public Test
     }
 
     const iox::RuntimeName_t m_processname{"TestProcess"};
-    const pid_t m_pid{42U};
+    const uint32_t m_pid{42U};
```

Comment:
Same question as before (why required and do we understand the implications?).

Reply:
`pid` is in this file only needed for the `ProcessManager::registerProcess` call which requires an `uint32_t`.